src/flint/fmpz.jl

```diff
@@ -1744,6 +1744,20 @@ function rand_bits(::FlintIntegerRing, b::Int)
    return z
 end
 
+@doc Markdown.doc"""
+    rand_prime(::FlintIntegerRing, n::Int, proved::Bool=true)
+> Returns a random prime number with the given number of bits. If only a
+> probable prime is required, one can pass `proved=false`.
+"""
+function rand_prime(::FlintIntegerRing, n::Int, proved::Bool = true,
+			          rnd = _flint_rand_states[Threads.threadid()])
```

Comment:
Maybe also `rnd` as a keyword?

Reply:
In the end I decided to put it inside the function call and remove that argument altogether, like the previous function (rand_bits). This makes more sense actually, as it's not ever going to be set to anything else.